wiz/README.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Wiz Buildkite Plugin Template
+description: Utilize the Wiz Buildkite Plugin for scanning in your Buildkite pipelines.
+author: Buildkite
+use_cases: ["Security", "CI"]
+tools: ["Wiz"]
+primary_emojis: [":wiz:", ":buildkite:"]
+platforms: []
+languages: []
+---
+
+# Wiz Buildkite Plugin Template
+
+This template provides a continuous integration (CI) pipeline that leverages the wiz-buildkite-plugin to perform security scans.
+
+
+## How it works
+
+This template:
+
+1. Configures the Wiz plugin.
+2. Executes the specified type of scan.
+
+## Usage
+
+To use this plugin, add it to your pipeline.yml:
```

Comment:
This part isn't necessary since the template _is_ the pipeline.

Reply:
done